src/Microsoft.SqlTools.ServiceLayer/SqlProjects/Contracts/Projects/GetCrossPlatformCompatibility.cs

```diff
@@ -0,0 +1,28 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+#nullable disable
+
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+using Microsoft.SqlTools.ServiceLayer.Utility;
+
+namespace Microsoft.SqlTools.ServiceLayer.SqlProjects.Contracts
+{
+    public class GetCrossPlatformCompatiblityRequest
+    {
+        public static readonly RequestType<SqlProjectParams, ResultStatus> Type = RequestType<SqlProjectParams, ResultStatus>.Create("sqlProjects/getCrossPlatformCompatibility");
```

Comment:
should this have `GetCrossPlatformCompatiblityResult` instead of ResultStatus?

Reply:
That's what I had originally, but the handler registration didn't like it; I couldn't figure out why - must've missed something small somewhere - but doing this and casting back seems to be working.  I still want to figure that out, though.

Reply:
Yup, fresh eyes found it.